tests/app/main/views/test_dashboard.py

```diff
@@ -1333,3 +1336,81 @@ def test_should_show_all_jobs_with_valid_statuses(
         'ready to send',
         'sent to dvla'
     })
+
+
+def test_org_breadcrumbs_do_not_show_if_service_has_no_org(
+    client_request,
+    mock_get_template_statistics,
+    mock_get_service_templates_when_no_templates_exist,
+    mock_get_jobs,
+):
+    page = client_request.get('main.service_dashboard', service_id=SERVICE_ONE_ID)
+
+    assert not page.select('.navigation-breadcrumb')
+
+
+def test_org_breadcrumbs_do_not_show_if_user_is_not_an_org_member(
+    mocker,
+    client,
+    mock_get_template_statistics,
+    mock_get_service_templates_when_no_templates_exist,
+    mock_get_jobs,
+    active_caseworking_user,
+):
+    # active_caseworking_user is not an org member
+
+    service_one_json = service_json(SERVICE_ONE_ID,
+                                    users=[active_caseworking_user['id']],
+                                    organisation_id=ORGANISATION_ID)
+    mocker.patch('app.service_api_client.get_service', return_value={'data': service_one_json})
+
+    client.login(active_caseworking_user, mocker, service_one_json)
```

Comment:
You don’t have to stop using `client_request` in order to log in as a different user:
```suggestion
    client_request.login(active_caseworking_user, service=service_one_json)
```

Reply:
Didn't know about that - this test and the one below now use `client_request`